specification/resources/resource-manager/Microsoft.Resources/stable/2024-11-01/resources.json

```diff
@@ -6845,6 +6853,18 @@
           }
         }
       }
+    },
+    "ValidationLevel": {
+      "type": "string",
+      "description": "The level of validation performed on the deployment. This value can be either Template, Provider, or ProviderWithoutWritePermissionChecks. At Template level, static analysis and validation of the template is performed. At Provider level, static analysis of the template is performed and resource declarations are sent to resource providers for semantic validation. At ProviderWithoutWritePermissionChecks level, static analysis of the template is performed and resource declarations are sent to resource providers for semantic validation without checking for write permissions.",
+      "enum": [
+        "Template",
+        "Provider",
+        "ProviderWithoutWritePermissionChecks"
```

Comment:
While this name is very descriptive, it's quite long. Might be worth seeing if anyone has other suggestions.